### PR TITLE
Apply design system

### PR DESCRIPTION
**Pull request by joelcarr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,118 +1,44 @@
-# Barnardos consent form generator
+# consent-form-builder
```
Comment (kittysquee) [inline README.md]: This seems like a regression from the previous title.

File: README.md [hunk 2]
```diff
@@ -1,118 +1,44 @@
...
 initially be paper signed by a participant, and later be digitally signed.
 
-## Developer guide
+## Getting started
 
-The following sections of the readme outline the software stack used to create the application,
-how to get the application running and how to make and submit changes.
+Install:
 
-For this repo, we are using [prettier](https://prettier.io/) to handle formatting of our JS / json / Markdown / scss files.
```
Comment (kittysquee) [inline README.md]: We're still using Prettier? Do we want to leave this in?

File: README.md [hunk 3]
```diff
@@ -1,118 +1,44 @@
...
 The purpose of the consent form generator is to allow a Barnardos researcher to
 enter details about an upcoming research session and produce consent forms that can
 initially be paper signed by a participant, and later be digitally signed.
 
-## Developer guide
+## Getting started
 
-The following sections of the readme outline the software stack used to create the application,
-how to get the application running and how to make and submit changes.
+Install:
```
Comment (kittysquee) [inline README.md]: Do we think that someone new to a Ruby/Rails project would be able to set this up from this guide?

File: README.md [hunk 4]
```diff
@@ -1,118 +1,44 @@
...
+    * `CREATE USER consent;`
+  * Add a database named 'consent' to your postgres instance, and make 'consent' the owner
+    * `CREATE DATABASE consent OWNER consent;`
+  * Add a database named 'consent_test' to your postgres instance, and make 'consent' the owner
+    * `CREATE DATABASE consent_test OWNER consent_test;`
 * Update the database schema using `bundle exec rails db:migrate`
-* If you have issues building your test database (`consent_test`), upgrade user to a SUPERUSER, access PostgreSQL in your terminal (`psql postgres`), then `ALTER USER consent WITH SUPERUSER;`. To check user attributes `\du`. Quit `\q`
-
-You can now start the rails server.
+* `hivemind Procfile.dev` - start a rails server with
```
Comment (kittysquee) [inline README.md]: Start a rails server with `hivemind Procfile.dev` may sound a little clearer.

**Comment by kittysquee** [comment]
screen shot 2018-06-06 at 15 32 05

screen shot 2018-06-06 at 15 34 03

screen shot 2018-06-06 at 15 35 53

screen shot 2018-06-06 at 15 37 25

screen shot 2018-06-06 at 15 38 16

screen shot 2018-06-06 at 15 48 49

screen shot 2018-06-06 at 15 50 03

**Comment by kittysquee** [comment]
I haven't had a chance to go through the entire PR. I will provide more feedback on Monday 😸

File: README.md (hunk 1 above)
Comment (joelcarr) [reply README.md]: The aim of these `README.md` updates was to bring it inline with our other repos' `README.md`s. Our other repos have the repo name as the heading for the document.

File: README.md (hunk 4 above)
Comment (joelcarr) [reply README.md]: For now, I am going to update this to read:

File: app/views/research_sessions/questions/researcher.html.erb [hunk 5]
```diff
@@ -1,44 +1,108 @@
...
-          </div>
-        </div>
-
-      </fieldset>
-
-      <%= render 'button_bar' %>
+<%= c('Step') do %>
+  <%= c('Title', text: 'Researcher') %>
+  <%= c('Paragraph') do %>
+    The researcher’s details will go directly on the sheet sent to participants of this session so they can contact you with any queries. The researcher should refer to the safeguarding code of conduct in <%= c('Link', src: 'http://b-hive.barnardos.org.uk/Interact/Pages/Content/Document.aspx?id=5100&SearchId=', text:'BREC Guidance') %> so they are confident they know what to do if a safeguarding concern arises during the session.
```
Comment (kittysquee) [inline app/views/research_sessions/questions/researcher.html.erb]: Should this be in the en.yml?

File: app/views/research_sessions/questions/researcher.html.erb [hunk 6]
```diff
@@ -1,44 +1,108 @@
...
   <% end %>
-</div>
+
+  <%= c('HeadingOne', text: 'Who will the participants meet as part of the session?') %>
+  <%= c('Paragraph') do %>
+    Who is the person who is leading the research or participation session?
```
Comment (kittysquee) [inline app/views/research_sessions/questions/researcher.html.erb]: en.yml

File: features/aids_for_devops.feature [hunk 7]
```diff
@@ -1,3 +1,4 @@
+@wip
```
Comment (kittysquee) [inline features/aids_for_devops.feature]: Should we make this not a `@wip`?

File: features/create_a_copy.feature [hunk 8]
```diff
@@ -3,6 +3,7 @@ Feature: Create a copy
   I want to create a new session from an existing session
   So that I can avoid typing everything from scratch again
 
+@wip
```
Comment (kittysquee) [inline features/create_a_copy.feature]: Should we make this not a `@wip`?

File: features/printed_consent_forms.feature [hunk 9]
```diff
@@ -1,3 +1,4 @@
+@wip
```
Comment (kittysquee) [inline features/printed_consent_forms.feature]: Should we make this not a `@wip`?

File: frontend/components/Header/_index.html.erb [hunk 10]
```diff
@@ -0,0 +1,3 @@
+<header class="Header">
+  <%= image_tag asset_pack_path('components/Header/static/logo.svg'), class: 'Header-logo', alt: 'Barnardo’s, Believe in children' %>
```
Comment (kittysquee) [inline frontend/components/Header/_index.html.erb]: Should this text be in the `en.yml`?

Comment (joelcarr) [reply frontend/components/Header/_index.html.erb]: It should be, there is a lot of text that should be moved to `en.yml`. The [current alt text](https://github.com/barnardos/consent-form-builder/blob/master/app/views/layouts/application.html.erb#L29) is also hardcoded in the view.

**Pull request merged**